app/addons/documents/index-editor/actions.js

```diff
@@ -127,7 +127,7 @@ function deleteView (options) {
 
     // if the user was on the index that was just deleted, redirect them back to all docs
     if (options.isOnIndex) {
-      var url = FauxtonAPI.urls('allDocs', 'app', options.database.safeID(), '?limit=' + FauxtonAPI.constants.DATABASES.DOCUMENT_LIMIT);
```

Comment:
You have removed the limit here. Are we adding it in elsewhere? Or don't we need it?

Reply:
It's not needed because the query is limited by the num of docs per page.